phoenix-core/src/main/java/org/apache/phoenix/coprocessor/GlobalIndexRegionScanner.java

```diff
@@ -408,7 +413,7 @@ public void close() throws IOException {
             try {
                 if (verificationResultRepository != null) {
                     verificationResultRepository.logToIndexToolResultTable(verificationResult,
-                            verifyType, region.getRegionInfo().getRegionName(), skipped);
+                            verifyType, region.getRegionInfo().getRegionName(), skipped, shouldRetry);
```

Comment:
Can we add a log here. It will be helpful to know the value of shouldRetry.

Reply:
Wouln't it be too chatty? I added logging for the true case already. I think it should be enough